Information Sharing/Deploying a Shiny App.md

```diff
@@ -0,0 +1,128 @@
+# Deploying (Publishing) a Shiny App
+
+At PHS, we currently use [shinyapps.io](http://shinyapps.io) to host Shiny apps. The server is located outside of the UK and hosted by Posit, which means that we can only host public-facing apps with no confidential data. If you have doubts about this for your app, you should contact the PHS Stats Gov [phs.statsgov@phs.scot](mailto:phs.statsgov@phs.scot) team to assess. General [dashboard development guidance](https://public-health-scotland.github.io/knowledge-base/docs/Information%20Sharing?doc=Dashboard%20development%20guidance.md) is available and should be followed when developing apps, prior to deployment.
+
+> [!NOTE]
+> There is work underway to provide a platform to host Shiny apps and another information products that may use confidential data and allow for greater control over who and what can be viewed. This will use the Posit Connect software and is being managed by the Posit Programme.
+
+## Contents
+
+- [Deploying (Publishing) a Shiny App](#deploying-publishing-a-shiny-app)
+  - [Contents](#contents)
+  - [General Guidance](#general-guidance)
+  - [Deploying a Shiny App](#deploying-a-shiny-app)
+    - [Adding Authentication to Apps](#adding-authentication-to-apps)
+    - [Troubleshooting](#troubleshooting)
+  - [Deleting a Shiny App](#deleting-a-shiny-app)
+
+## General Guidance
+
+There are some key points to consider before deploying an app:
+
+- **Data**: Ensure that the data used in the app is not confidential and can be shared publicly. If you are unsure, contact the [PHS Stats Gov team](mailto:phs.statsgov@phs.scot).
```

Comment:
I think it needs to be made extra clear here that data can't be confi whether it is available for users to view - or even stored in the background at any point.

Reply:
Agreed, updated with something more specific.